x-pack/plugins/ml/common/types/capabilities.ts

```diff
@@ -78,7 +78,7 @@ export function getPluginPrivileges() {
   const adminMlCapabilitiesKeys = Object.keys(adminMlCapabilities);
   const allMlCapabilitiesKeys = [...adminMlCapabilitiesKeys, ...userMlCapabilitiesKeys];
   // TODO: include ML in base privileges for the `8.0` release: https://github.com/elastic/kibana/issues/71422
-  const savedObjects = ['index-pattern', 'dashboard', 'search', 'visualization'];
+  const savedObjects = ['index-pattern', 'dashboard', 'search', 'visualization', 'ml-job'];
```

Comment:
@legrego should read/all access to `ml-job` also be given to `apmUser`?

Reply:
I think they'd just need `read` access to these job definitions, but I haven't taken a deep dive into this PR just yet.